### ISSUE TITLE
Generate the training/test set of CUHK03 according to AlignedReid paper

### ISSUE THREAD

**Issue by ZHHJemotion** [huanghoujing/AlignedReID-Re-Production-Pytorch#41]
Hi Houjing,

I used this code to train with CUHK03 dataset based on the new protocol as you wrote in the readme, it performed good! But now I want to split CUHK03 dataset to training/test set based on what paper wrote, I read the code and found the information of spliting has been saved in the file of "re_ranking_train_test_split.pkl", could you please give me some advices to generate this information file for spliting?  Thanks a lot!!!

**Comment by huanghoujing** [comment]
Thanks for your attention.
For your need, when transforming CUHK03 using `script/dataset/transform_cuhk03.py`, you can provide your own `train_test_partition_file` corresponding to the paper's split. This `train_test_partition_file` specifies image names for `train`, `query` and `gallery` sets. In fact, I am not sure about the detail split of the paper and sorry for that.

**Comment by ZHHJemotion** [comment]
@huanghoujing Thanks a lot for your reply. I would ask the author for some details about the split. Thanks again.

**Comment by ZHHJemotion** [comment]
@huanghoujing I have generated the training/test set based on the split strategy in AlignedReID paper by myself: 1267 ids for training and 200 ids for testing. Finally, I got the rank-1 acc after re-ranking with mutual learning of 94.22%  and without mutual learning of 92.41%, they are a litter worse than in paper.

**Comment by huanghoujing** [comment]
Well, I am impressed by the 92%+ performance! You provide some reference for later similar experiments. Thanks a lot.

**Comment by ZHHJemotion** [comment]
Thank you for your reply. How can I provide these reference?

**Comment by huanghoujing** [comment]
Oh, I may not express it well. I was not asking you to provide anything. I just meant that your comments here may be useful for people doing the same experiments. Forgive me for my expression.

**Comment by ZHHJemotion** [comment]
I forked it and added the code of new split training/testing set, so that someone can reference. Thank you!
